### ISSUE TITLE
Chrome doesn't show the context menu to inspect components

### ISSUE THREAD

**Issue by realtebo** [vuejs/vue-devtools#1194]
### Version
5.3.3

### Browser and OS info
Chrome  81 on Windows 10 64bit 19.09

### Steps to reproduce
I have a page made using vue 2.x (latest),  I right -click and I don't see inspect vue components.

### What is expected?
 I right -click and I see inspect vue components.

### What is actually happening?
 I right -click and I don't see inspect vue components.

---
Vue devtolls is enable for all pages and my page is build for dev, not for production

**Comment by lolPlatinumPlayer** [comment]
Maybe you can try chrome89.

I use win10 and vue devtools 5.3.4 and vue2.6, I can see `inspect vue components`

**Comment by jbhamilton** [comment]
Same is happening for me but in Firefox : 

Version : 6.0.12

Browser : Firefox 97.0.1 (64bit) on Linux Kubuntu 18.04

**Comment by dylanett** [comment]
My context menu hasn't worked since the update to V6 on Firefox either.  Note that you can go to the Components tab, click the My Location icon, then click on any component to inspect it.
![image](https://user-images.githubusercontent.com/17145970/155197224-7d502cdc-cad2-47f7-9381-3353abebb05d.png)

**Comment by jbhamilton** [comment]
@dylanett Yep been using that since the update but it's a regression in my eyes.

**Comment by calebwaldner** [comment]
> @dylanett Yep been using that since the update but it's a regression in my eyes.

Agreed, missing the right click to inspect feature.

**Comment by Akryum** [comment]
This is not implemented yet in the new codebase.
